{"priority": "major", "kind": "bug", "repository": {"links": {"self": {"href": "data/repositories/ignitionrobotics/ign-rendering.json"}, "html": {"href": "#!/ignitionrobotics/ign-rendering"}, "avatar": {"href": "data/bytebucket.org/ravatar/{f6b404b6-f373-460d-a470-69ff6f27cd93}ts=1533308"}}, "type": "repository", "name": "ign-rendering", "full_name": "ignitionrobotics/ign-rendering", "uuid": "{f6b404b6-f373-460d-a470-69ff6f27cd93}"}, "links": {"attachments": {"href": "data/repositories/ignitionrobotics/ign-rendering/issues/45/attachments_page=1.json"}, "self": {"href": "data/repositories/ignitionrobotics/ign-rendering/issues/45.json"}, "watch": {"href": "https://api.bitbucket.org/2.0/repositories/ignitionrobotics/ign-rendering/issues/45/watch"}, "comments": {"href": "data/repositories/ignitionrobotics/ign-rendering/issues/45/comments_page=1.json"}, "html": {"href": "#!/ignitionrobotics/ign-rendering/issues/45/unloading-engine-plugin-causes-tests-to"}, "vote": {"href": "https://api.bitbucket.org/2.0/repositories/ignitionrobotics/ign-rendering/issues/45/vote"}}, "reporter": {"display_name": "Ian Chen", "uuid": "{eaa6fca5-6deb-43f6-907f-971c144735dd}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Beaa6fca5-6deb-43f6-907f-971c144735dd%7D"}, "html": {"href": "https://bitbucket.org/%7Beaa6fca5-6deb-43f6-907f-971c144735dd%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:10b01d41-a2e9-4a41-a907-e6e2f03b6cd5/1e4adcdf-1946-4280-9aea-eb5f15a7f912/128"}}, "nickname": "Ian Chen", "type": "user", "account_id": "557058:10b01d41-a2e9-4a41-a907-e6e2f03b6cd5"}, "title": "Unloading engine plugin causes tests to crash on Windows", "component": null, "votes": 0, "watches": 1, "content": {"raw": "# Description \r\n\r\nOn windows, the tests run fine and calls to [unloadEnginePlugin](#!/ignitionrobotics/ign-rendering/src/8bfca2dce901c5bff2340c55006bdb52610a4a95/src/RenderEngineManager.cc#lines-397) return without an error but the tests crash on exit with the following msg\r\n\r\n```\r\nunknown file\r\nSEH exception with code 0xc0000005 thrown in the test body.\r\n```\r\nNote that this problem is observed on our windows jenkins machine that currently only run tests using the `ogre` rendering engine\r\n\r\n# Steps to Reproduce\r\n\r\n1. Run any ignition rendering test\r\n\r\n**Expected behavior:**  \r\n\r\nTest passes\r\n\r\n**Actual behavior:**\r\n\r\nTest crashes with error msg\r\n\r\n**Reproduces how often:** \r\n\r\nAll the time\r\n\r\n# Versions\r\n\r\nAll versions", "markup": "markdown", "html": "<h1 id=\"markdown-header-description\">Description</h1>\n<p>On windows, the tests run fine and calls to <a data-is-external-link=\"true\" href=\"#!/ignitionrobotics/ign-rendering/src/8bfca2dce901c5bff2340c55006bdb52610a4a95/src/RenderEngineManager.cc#lines-397\" rel=\"nofollow\">unloadEnginePlugin</a> return without an error but the tests crash on exit with the following msg</p>\n<div class=\"codehilite\"><pre><span></span>unknown file\nSEH exception with code 0xc0000005 thrown in the test body.\n</pre></div>\n\n\n<p>Note that this problem is observed on our windows jenkins machine that currently only run tests using the <code>ogre</code> rendering engine</p>\n<h1 id=\"markdown-header-steps-to-reproduce\">Steps to Reproduce</h1>\n<ol>\n<li>Run any ignition rendering test</li>\n</ol>\n<p><strong>Expected behavior:</strong>  </p>\n<p>Test passes</p>\n<p><strong>Actual behavior:</strong></p>\n<p>Test crashes with error msg</p>\n<p><strong>Reproduces how often:</strong> </p>\n<p>All the time</p>\n<h1 id=\"markdown-header-versions\">Versions</h1>\n<p>All versions</p>", "type": "rendered"}, "assignee": null, "state": "new", "version": null, "edited_on": null, "created_on": "2019-03-08T00:39:14.436127+00:00", "milestone": null, "updated_on": "2019-03-08T00:39:43.879702+00:00", "type": "issue", "id": 45}